.NET/Microsoft.Recognizers.Text.NumberWithUnit/Parsers/BaseCurrencyParser.cs

```diff
@@ -73,6 +73,11 @@ private static bool CheckUnitsStringContains(string fractionUnitCode, string fra
 
         private string GetResolutionStr(object value)
         {
+            if (value == null)
```

Comment:
Please add a comment here: // Nothing to resolve. This happens when the entity is a currency name only (no numerical value).